applications/matter_bridge/src/bridge.zap

```diff
@@ -1,6 +1,6 @@
 {
   "fileFormat": 2,
-  "featureLevel": 99,
+  "featureLevel": 100,
```

Comment:
This might be problematic, can you use ZAP tool from this release?

Reply:
Fixed